src/mrpro/data/_kdata/KData.py

```diff
@@ -96,6 +124,17 @@ def from_file(
 
         acqinfo = AcqInfo.from_ismrmrd_acquisitions(acquisitions)
 
+        for i in (5, 6):
+            user_indices = getattr(acqinfo.idx, f'user{i}', [])
+            for j in user_indices:
```

Comment:
don't check every label (time consuming) but do something like if len(torch.unique(user_indices)) > 1

Reply:
Also, just write idx.user5 and idx.user6 explicitly instead of constructing a string and using getattr.